<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WithdrawFunds</class>
 <widget class="QMainWindow" name="WithdrawFunds">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>780</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/sources/wallpapersden.com_ice-vs-fire-dragon-fight_1920x1080.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="sources.qrc">:/images/sources/Dragons.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>0</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1043</x>
      <y>15</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="sources.qrc">:/images/sources/Clock.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>911</width>
      <height>371</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
  	border-radius: 50px;
  	min-height: 20px;
 	min-width: 20px;
	border-image: url(:/images/sources/Back.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>380</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	border: 0;
}</string>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Algerian</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(82, 137, 225);
	border-radius: 25px;
	border-style:solid;
	border-width:3px;
	border-color: rgb(0, 0, 0);
}

QPushButton:hover {
	background-color: rgb(62, 117, 225);
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: rgb(42, 157, 225);
	border-style:solid;
	border-width:2px;
}</string>
     </property>
     <property name="text">
      <string> 50</string>
     </property>
     <property name="icon">
      <iconset resource="sources.qrc">
       <normaloff>:/images/sources/Hryvnia.png</normaloff>:/images/sources/Hryvnia.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>-54</x>
       <y>6</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/sources/Krilo2V1.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>7</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/sources/KrilaV3.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>-19</x>
       <y>18</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/sources/DragonTail.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_10</zorder>
    <zorder>label_8</zorder>
    <zorder>label_9</zorder>
    <zorder>pushButton_2</zorder>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>460</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	border: 0;
}</string>
    </property>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Algerian</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(82, 137, 225);
	border-radius: 25px;
	border-style:solid;
	border-width:3px;
	border-color: rgb(0, 0, 0);
}

QPushButton:hover {
	background-color: rgb(62, 117, 225);
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: rgb(42, 157, 225);
	border-style:solid;
	border-width:2px;
}</string>
     </property>
     <property name="text">
      <string> 100</string>
     </property>
     <property name="icon">
      <iconset resource="sources.qrc">
       <normaloff>:/images/sources/Hryvnia.png</normaloff>:/images/sources/Hryvnia.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>-54</x>
       <y>6</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/sources/Krilo2V1.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>18</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/sources/DragonTail.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>6</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/sources/KrilaV3.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_12</zorder>
    <zorder>label_13</zorder>
    <zorder>label_11</zorder>
    <zorder>pushButton_3</zorder>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>590</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	border: 0;
}</string>
    </property>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Algerian</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(82, 137, 225);
	border-radius: 25px;
	border-style:solid;
	border-width:3px;
	border-color: rgb(0, 0, 0);
}

QPushButton:hover {
	background-color: rgb(62, 117, 225);
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: rgb(42, 157, 225);
	border-style:solid;
	border-width:2px;
}</string>
     </property>
     <property name="text">
      <string> 200</string>
     </property>
     <property name="icon">
      <iconset resource="sources.qrc">
       <normaloff>:/images/sources/Hryvnia.png</normaloff>:/images/sources/Hryvnia.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>-54</x>
       <y>6</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/sources/Krilo2V1.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>6</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/sources/KrilaV3.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>18</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/sources/DragonTail.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_21</zorder>
    <zorder>label_22</zorder>
    <zorder>label_20</zorder>
    <zorder>pushButton_4</zorder>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>590</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	border: 0;
}</string>
    </property>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Algerian</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(82, 137, 225);
	border-radius: 25px;
	border-style:solid;
	border-width:3px;
	border-color: rgb(0, 0, 0);
}

QPushButton:hover {
	background-color: rgb(62, 117, 225);
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: rgb(42, 157, 225);
	border-style:solid;
	border-width:2px;
}</string>
     </property>
     <property name="text">
      <string> 500</string>
     </property>
     <property name="icon">
      <iconset resource="sources.qrc">
       <normaloff>:/images/sources/Hryvnia.png</normaloff>:/images/sources/Hryvnia.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>-54</x>
       <y>6</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/sources/Krilo2V1.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>6</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/sources/KrilaV3.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>18</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/sources/DragonTail.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_18</zorder>
    <zorder>label_19</zorder>
    <zorder>label_17</zorder>
    <zorder>pushButton_5</zorder>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>460</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	border: 0;
}</string>
    </property>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Algerian</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(82, 137, 225);
	border-radius: 25px;
	border-style:solid;
	border-width:3px;
	border-color: rgb(0, 0, 0);
}

QPushButton:hover {
	background-color: rgb(62, 117, 225);
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: rgb(42, 157, 225);
	border-style:solid;
	border-width:2px;
}</string>
     </property>
     <property name="text">
      <string> 1000</string>
     </property>
     <property name="icon">
      <iconset resource="sources.qrc">
       <normaloff>:/images/sources/Hryvnia.png</normaloff>:/images/sources/Hryvnia.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>-54</x>
       <y>6</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/sources/Krilo2V1.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>18</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/sources/DragonTail.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>6</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/sources/KrilaV3.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_16</zorder>
    <zorder>label_15</zorder>
    <zorder>label_14</zorder>
    <zorder>pushButton_6</zorder>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>430</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border-radius: 10px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>370</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Own amount of money</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Popular amount of money</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_7">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>540</y>
      <width>321</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	border: 0;
}</string>
    </property>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Algerian</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(255, 68, 35);
	border-radius: 25px;
	border-style:solid;
	border-width:3px;
	border-color: rgb(0, 0, 0);
}

QPushButton:hover {
	background-color: rgb(235, 48, 35);
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: rgb(215, 88, 35);
	border-style:solid;
	border-width:2px;
}</string>
     </property>
     <property name="text">
      <string>Withdraw money</string>
     </property>
     <property name="icon">
      <iconset resource="sources.qrc">
       <normaloff>:/images/sources/Money.png</normaloff>:/images/sources/Money.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>-54</x>
       <y>6</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/sources/KriloKrasnoe2.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>164</x>
       <y>6</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/sources/KriloKrasnoe.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>18</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/sources/DragonTail.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_25</zorder>
    <zorder>label_24</zorder>
    <zorder>label_23</zorder>
    <zorder>pushButton_7</zorder>
   </widget>
   <widget class="QFrame" name="frame_8">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	border: 0;
}</string>
    </property>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Algerian</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(82, 137, 225);
	border-radius: 15px;
}

QPushButton:hover {
	background-color: rgb(62, 117, 225);
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: rgb(42, 157, 225);
	border-style:solid;
	border-width:2px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="sources.qrc">
       <normaloff>:/images/sources/BackA.png</normaloff>:/images/sources/BackA.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="sources.qrc"/>
 </resources>
 <connections/>
</ui>
